client/app/components/queries/AddToDashboardDialog.jsx

```diff
@@ -52,9 +52,9 @@ function AddToDashboardDialog({ dialog, visualization }) {
         notification.success(
           "Widget added to dashboard",
           <React.Fragment>
-            <Link href={`${dashboard.url}`} onClick={() => notification.close(key)}>
+            <Button type="link" href={`${dashboard.url}`} onClick={() => notification.close(key)}>
```

Comment:
I'm not 100% sure about semantics of this component. On the one hand, it's link, on the other - it's button. But for me it looks that link role is primary (as if you click it - it will bring you to a dashboard page), and onClick action is secondary (just to hide irrelevant UI). So my suggestion is to revert it to `Link`